src/PowerShell/PowerShellManager.cs

```diff
@@ -92,7 +92,7 @@ internal void InvokeProfile(string profilePath)
             Exception exception = null;
             if (profilePath == null)
             {
-                _logger.Log(LogLevel.Trace, string.Format(PowerShellWorkerStrings.FileNotFound, "profile.ps1", FunctionLoader.FunctionAppRootPath));
```

Comment:
Do we still need the `_logger` variable anymore?

Reply:
We have another logging call below that writes user logs, so yes, we still need `_logger`.